web/wp-content/themes/lfevents/library/lfe-functions.php

```diff
@@ -51,6 +51,12 @@ function lfe_get_related_events( $parent_id ) {
 	} else {
 		$term = wp_get_post_terms( $parent_id, 'lfevent-category', array( 'fields' => 'ids' ) );
 
+		if ( empty( ! $term ) ) {
```

Comment:
😅 PHP fixed

I've tightened up the menu height. I think I may have checked it with Admin bar showing and added an extra 2em. 

Ideally when the alert bar is present it would add a class to body so that other elements relying on top being 0 can be moved down to make way for it, but when I implemented that it wasn't working, I think either due to the placement of the alert bar function or the way FoundationPress handles body classes. So as it is currently, the extra top padding is applied regardless of the presence of the alert bar. If there are other areas where the alert bar now affects positioning adding a body class would be worthwhile.

New spacing:  

![20200420104658-Venue   Travel | Linux Foundation Events](https://user-images.githubusercontent.com/10615884/79738220-610b8f00-82f4-11ea-8436-ba398295c538.jpg)

The question of "too much space" is, I guess, very subjective. In the first round, one could say the space is equidistant to that of the logo and 1st nav item in the menu, bringing some symmetry. Adding space generally improves a design and allows users to digest it easier. But it's not a hill I wanna die on, so am happy to defer to your judgement! 😂

Reply:
Occasionally (esp on the China site) we have a side menu that is so long it goes below the fold so I don't want more space up there than we need.  I like how you've got it now though.